i18n/qqq.json

```diff
@@ -8,5 +8,10 @@
 	"github-desc": "{{desc|name=GitHub|url=https://www.mediawiki.org/wiki/Extension:GitHub}}",
 	"github-par-filename": "Label for the contents of the \"file name\" parameter\n{{Identical|Filename}}",
 	"github-par-branchname": "Label for the contents of the \"branch name\" parameter",
-	"github-par-reponame": "Label for the contents of the \"repository name\" parameter"
+	"github-par-reponame": "Label for the contents of the \"repository name\" parameter",
+	"github-par-lang": "Label for the contents of the \"lang\" parameter",
+	"github-par-line": "Label for the contents of the \"line\" parameter",
+	"github-par-start": "Label for the contents of the \"start\" parameter",
+	"github-par-highlight": "Label for the contents of the \"highlight\" parameter",
+	"github-par-inline": "Label for the contents of the \"inline\" parameter",
```

Comment:
These system messages got introduced with version [1.0.0](https://github.com/JeroenDeDauw/GitHub/blob/439ca62d6b6f9615f67f73873da138e6fcccb1be/src/Setup.php#L100) but I am not sure how the were supposed to integrate, i.e. the name of the parser function supposed to display the stuff.

The messages added to "qqq.json" are shown to the translators of the system messages to provide assistance and context in understanding what the programmer actually meant with them. If you introduce system messages here you also need to add them in "en.json" which is the source file for system messages.